### DISCUSSION TITLE
Hero 13 Screen Time Out

### DISCUSSION THREAD

**Discussion by TJ311** [gopro/labs#1094] [Feature Requests]
Is it possible to have a feature that allows me to set the front screen time out separately from the rear screen? It was an option on the hero 12 but not on the Hero 13. I like to have my rear screen shut off right away, and the front screen stay on for 5 minutes. Thanks

**Comment by dnewman-gpsw** [comment]
Turns out the QR commands for HERO12 still work on HERO13.

This is no rear LCD off immediately setting, but it can be set for 1 minute, and the front set to 5 minute. Try this:

![image](https://github.com/user-attachments/assets/0ee0f4e2-0d49-4df3-8a22-9c9e80d273bf)

**Reply by TJ311** [reply]
Worked perfectly! Thank you so much.
